schema/corsika_observation_level.schema.yml

```diff
@@ -0,0 +1,36 @@
+%YAML 1.2
+---
+title: Schema for corsika_observation_level model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/data.metaschema.yml
+meta_schema_version: 0.1.0
+name: corsika_observation_level
+description: |-
+  Observation altitude above see level assumed in the air-shower code.
+  This should be lower than the altitude of the lowest array element.
+short_description: Observation altitude above see level assumed in the air-shower code.
+data:
+  - type: double
+    units: m
+    allowed_range:
+      min: 0.0
+instrument:
+  class: Site
+  type:
+    - Observatory
+  site:
+    - North
+    - South
+activity:
+  setting:
+    - SetParameterFromExternal
+    - SetArrayElementCoordinates
+  validation:
+    - ValidateParameterByExpert
+    - ValidateArrayElementCoordinates
+source:
+  - Initial instrument setup
+simulation_software:
+  - name: corsika
+    parameter_name: OBSLEV
```

Comment:
Agree and removed it - it is clearer in the meanwhile to me that we don't need it here: we will have a tool generating documentation and it will be straightforward for this tool to read the mapping of parameter names using the map coded in simtools.

Reply:
Also removed it from the schema, which is part of simtools (in the new-simulation model branch, [here](https://github.com/gammasim/simtools/blob/41d9ee163a385847415bb78af1f3788b3de61dd0/simtools/schemas/data.metaschema.yml#L55))